examples/wasmcam/main.go

```diff
@@ -22,7 +22,7 @@ import (
 var processFrameWasm []byte
 
 var (
-	device = flag.String("device", "/dev/video0", "video capture device")
+	device = flag.String("device", "0", ":video capture device")
```

Comment:
```suggestion
	device = flag.String("device", "0", "video capture device")
```

I think this is just an extra character that slipped in?

Reply:
yep, should be fixed now :)